setup.cfg

```diff
@@ -24,15 +24,6 @@ universal = 1
 test = pytest
 
 [flake8]
-exclude =
-	.git,
-	__pycache__,
-	build,
-	dist,
-	.tox,
-	venv
-max-complexity = 12
 per-file-ignores =
```

Comment:
i couldn't figure out how to move this config to .pre-ommit-config.yaml, all my attempts will cause the linter to fail.  do you happen to know how to do it @zxiiro ?

Reply:
Use an exclude block like this in pre-commit https://github.com/opendaylight/releng-builder/blob/master/.pre-commit-config.yaml#L27-L31 it only accepts regex if I recall so even the list must be in regex form.